src/main/resources/META-INF/resources/reflection-config.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "name" : "org.eclipsefoundation.geoip.client.model.Country",
```

Comment:
the class annontated with @RegisterForReflection don't need to be added here. They will be added by Quarkus at build time.